### PR TITLE
Set timezone for MAGWest

### PR DESCRIPTION
**Pull request by kitsuta**

### CODE REVIEW

**Comment by binary1230** [comment]
oh WOW nice catch

File: event-west.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ uber::config::year: 2017
 
 uber::config::event_venue: 'The Hyatt Regency Santa Clara'
 uber::config::event_venue_address: '5101 Great America Pkwy, Santa Clara, CA 95054'
+uber::config:event_timezone: 'US/Pacific'
```
Comment (RobRuana) [inline event-west.yaml]: Is this missing a colon between "config" and "event_timezone"?

**Review by RobRuana — CHANGES_REQUESTED** [review]
Needs two colons between "config" and "event_timezone"

File: event-west.yaml (hunk 1 above)
Comment (kitsuta) [inline event-west.yaml]: ;___; yup

**Review by RobRuana — APPROVED** [review]
I have no idea what this controls or if it's even in the right format, but I trust you!

**Comment by kitsuta** [comment]
Haha, it controls display of times in the app. As for format, it's the pytz timezone format -- all of the options are listed here: https://stackoverflow.com/questions/13866926/python-pytz-list-of-timezones

**Pull request merged**